openapi.yaml

```diff
@@ -279,6 +282,40 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/GenericApiResponse"
+  /getSchemaRegEnvs:
+    get:
+      operationId: schemaRegEnvsGet
+      summary: get schema registry environments
+      tags:
+        - SchemaRequest
+      responses:
+        "200":
```

Comment:
Question: I notice that we only document the `200` response for our endpoints. Are there no specific errors returned by the API when a call fails?

Reply:
It's all I could get from the current API. I think we will describe the responses more specific when we update parts of the API and use the open API spec more strict (BE and FE have to use it etc.).